### PR TITLE
Add 0.5.4 release notes

### PR DESCRIPTION
**Pull request by jni**

### CODE REVIEW

File: docs/release/release_0_5_4.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+- Speed up `get_status` for Shapes layer by using bounding boxes ([#7144](https://github.com/napari/napari/pull/7144))
+- Enh: Add zoom options and keybindings to View menu ([#7200](https://github.com/napari/napari/pull/7200))
+- [ux/ui] Put blending controls under opacity slider in layer controls ([#7202](https://github.com/napari/napari/pull/7202))
+- Enable conflicts detection with layer keybindings when changing viewer ones and add group info to conflicts popup ([#7231](https://github.com/napari/napari/pull/7231))
+- Speedup `_is_convex` by avoid calling `np.roll` ([#7256](https://github.com/napari/napari/pull/7256))
+- Sort plugin manifests in alphabetical order for registration ([#7266](https://github.com/napari/napari/pull/7266))
+
+## Performance
+
+- Speed up `get_status` for Shapes layer by using bounding boxes ([#7144](https://github.com/napari/napari/pull/7144))
```
Comment (Czaki) [inline docs/release/release_0_5_4.md:53]: maybe also: `- Speedup `_is_convex` by avoid calling `np.roll` ([#7256](https://github.com/napari/napari/pull/7256))`

Comment (jni) [reply docs/release/release_0_5_4.md:53]: Yup, was missing a label. Will add it now. (Both the line and the label.)

**Comment by jni** [comment]
@Czaki thanks for looking this over? If it looks ok now, could I get an approval? That way I can merge once I've checked the build, and tag the release candidate. 🙏

**Review by Czaki — APPROVED** [review]

**Comment by Czaki** [comment]
What did you think about https://github.com/napari/napari/pull/7247 maybe add it to highlight?

**Comment by jni** [comment]
Gah... We are getting these errors on many of the gallery builds:

```
/home/circleci/project/docs/docs/conf.py:279: FutureWarning: `QApplication` instance access through `get_app` is deprecated and will be removed in 0.6.0.
Please use `get_qapp` instead.

  if app := napari.qt.get_app():
Traceback (most recent call last):
  File "/home/circleci/project/napari/napari/_qt/qt_main_window.py", line 237, in set_status_and_tooltip
    self._qt_viewer.viewer.status = status_and_tooltip[0]
  File "/home/circleci/project/napari/napari/utils/events/evented_model.py", line 307, in __setattr__
    with ComparisonDelayer(self):
  File "/home/circleci/project/napari/napari/utils/events/evented_model.py", line 520, in __exit__
    self._target._check_if_values_changed_and_emit_if_needed()
  File "/home/circleci/project/napari/napari/utils/events/evented_model.py", line 345, in _check_if_values_changed_and_emit_if_needed
    getattr(self.events, name)(value=new_value)
  File "/home/circleci/project/napari/napari/utils/events/event.py", line 764, in __call__
    self._invoke_callback(cb, event if pass_event else None)
  File "/home/circleci/project/napari/napari/utils/events/event.py", line 802, in _invoke_callback
    _handle_exception(
  File "/home/circleci/project/napari/napari/utils/events/event.py", line 789, in _invoke_callback
    cb(event)
  File "/home/circleci/project/napari/napari/_qt/qt_main_window.py", line 1572, in _status_changed
    self._status_bar.setStatusText(event.value)
  File "/home/circleci/project/napari/napari/_qt/qt_main_window.py", line 857, in _status_bar
    return self._qt_window.statusBar()
AttributeError: 'Window' object has no attribute '_qt_window'
```

They seem like real bugs. Too late to debug here, will look in the morning, but if anyone has time to investigate today in other time zones, it would be much appreciated! 😅 🙏 

I guess it comes from napari/napari#7269, but I guess we don't check for a no-warning docs build in the main repo?

CC @dalthviz 🙏

**Comment by Czaki** [comment]
I think that this PR needs to be merged https://github.com/napari/docs/pull/495.

Just updated to trigger build with main of napari

**Comment by Czaki** [comment]
Yes, it looks like #495 solve the problem

**Comment by jni** [comment]
Hmmm still getting this failure:

```pytb
Traceback (most recent call last):
  File "/home/circleci/project/napari/napari/_qt/qt_main_window.py", line 237, in set_status_and_tooltip
    self._qt_viewer.viewer.status = status_and_tooltip[0]
  File "/home/circleci/project/napari/napari/utils/events/evented_model.py", line 307, in __setattr__
    with ComparisonDelayer(self):
  File "/home/circleci/project/napari/napari/utils/events/evented_model.py", line 520, in __exit__
    self._target._check_if_values_changed_and_emit_if_needed()
  File "/home/circleci/project/napari/napari/utils/events/evented_model.py", line 345, in _check_if_values_changed_and_emit_if_needed
    getattr(self.events, name)(value=new_value)
  File "/home/circleci/project/napari/napari/utils/events/event.py", line 764, in __call__
    self._invoke_callback(cb, event if pass_event else None)
  File "/home/circleci/project/napari/napari/utils/events/event.py", line 802, in _invoke_callback
    _handle_exception(
  File "/home/circleci/project/napari/napari/utils/events/event.py", line 789, in _invoke_callback
    cb(event)
  File "/home/circleci/project/napari/napari/_qt/qt_main_window.py", line 1572, in _status_changed
    self._status_bar.setStatusText(event.value)
  File "/home/circleci/project/napari/napari/_qt/qt_main_window.py", line 857, in _status_bar
    return self._qt_window.statusBar()
AttributeError: 'Window' object has no attribute '_qt_window'
```

which does look like a genuine napari bug.

**Comment by jni** [comment]
Ok what. It was just the toc missing. That is a weird manifestation of the missing toc LOL.

**Pull request merged**